src/extension.ts

```diff
@@ -202,7 +204,7 @@ export async function activate(context) : Promise<extensionapi.ExtensionAPI> {
     subscriptions.forEach((element) => {
         context.subscriptions.push(element);
     }, this);
-
+ 
```

Comment:
Is this diff indicating there's a spurious space on the blank line?  I can't quite tell.

Reply:
I will remove it.